content/html/concepts/attributes/attributes.md

```diff
@@ -13,22 +13,39 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-Attributes can be added to the opening tag of an HTML element to change its default behavior or provide additional data about it.
+HTML attributes are data that can be added to the opening tag of an HTML element. These attributes modify the default behavior of the element or provide additional information about it. 
+Attributes consist of a name and are usually set equal to a value. Values are enclosed in double or single quotation marks. 
 
-Attributes consist of a name and a value in the format `name="value"`:
+## Syntax 
 
-```pseudo
-<tag name="value"></tag>
+The syntax for adding attributes to an HTML element is as follows: 
+
+ ```html 
+ <tagname attribute="value"></tagname> 
+ ```
+
+> **Note:** Attribute names are not case-sensitive, but it is good practice to write HTML markup in lowercase.
+ 
+## Examples 
+
+ ### Image Attributes
+
+Using an  `<img>` element, which is commonly used to display images, reveals several **attributes**, such as `src`, `width`, and `height`. These attributes exert significant influence over the manner in which the image appears on the web page. The dimensions of the element in particular, once specified, enable precise control over the end result.
```

Comment:
[proofreading] Always hyperlink elements that were written about. I highly recommend checking out the [Codecademy Doc's website](https://www.codecademy.com/resources/docs) to find them.  
```suggestion
Using an  [`<img>`](https://www.codecademy.com/resources/docs/html/elements/img) element, which is commonly used to display images, reveals several **attributes**, such as `src`, `width`, and `height`. These attributes exert significant influence over the manner in which the image appears on the web page. The dimensions of the particular element, once specified, enable precise control over the end result.
```